src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/altar/AncientPedestal.java

```diff
@@ -46,14 +54,17 @@
  * @see AncientAltarTask
  *
  */
-public class AncientPedestal extends SimpleSlimefunItem<BlockDispenseHandler> {
+public class AncientPedestal extends SimpleSlimefunItem<BlockDispenseHandler> implements HologramOwner {
```

Comment:
So I added this based on the fact that the Holographic Projector did the same, it has no purpose and it has no purpose in the Holographic Projector either, **I did use it previously** but do no longer, mainly just thought I would leave it in case its there as a tell that it utilizes a Armor Stand for a Hologram, again just purely based on how the holographic projector did

Reply:
fixed in latest commit